cmd/podman/inspect.go

```diff
@@ -122,13 +123,18 @@ func iterateInput(ctx context.Context, c *cli.Context, args []string, runtime *l
 				inspectError = errors.Wrapf(err, "error getting libpod container inspect data %s", ctr.ID())
 				break
 			}
-			data, err = shared.GetCtrInspectInfo(ctr, libpodInspectData)
+			artifact, err := getArtifact(ctr)
+			if inspectError != nil {
+				inspectError = err
+				break
+			}
+			data, err = shared.GetCtrInspectInfo(ctr.Config(), libpodInspectData, artifact)
```

Comment:
"shared" between whom? Seems like a very questionable package name.

Reply:
shared between "things" ... this has been around for a while. not part of this PR